tools/tests/vrt/backstop-example-settings.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * This example demonstrates how to create settings for a given testing scenario
+ * for backstopjs. By replicating this setup, one can create multiple types of
+ * tests and test groups for running Backstop VRT.
+ */
+
+module.exports = () => {
+  // Set which resolutions to take screenshots at.
+  const viewports = [
+    {
+      name: 'desktop',
+      width: 1401,
+      height: 1500,
+    },
+    {
+      name: 'mobile',
+      width: 720,
+      height: 1280,
+    },
+  ];
+
+  // Set the defaults for all of the screenshots. This can be used to powerfully
+  // modify the page before capturing.
+  const defaultScenarioSettings = {
+    hideSelectors: [],
+    removeSelectors: [],
+    selectorExpansion: true,
+    selectors: [],
+    readyEvent: null,
+    delay: 500,
+    misMatchThreshold: 0.1,
+    requireSameDimensions: true,
+    onBeforeScript: '',
+    onReadyScript: '',
+  };
+
+  // Set which pages to capture, and how to label them.
+  const scenarios = [
+    {
+      label: 'article',
+      url: 'http://0.0.0.0:8080/pl/?p=pages-article',
+      ...defaultScenarioSettings,
+    },
+    {
+      label: 'homepage',
+      url: 'http://0.0.0.0:8080/pl/?p=pages-homepage',
+      ...defaultScenarioSettings,
+    },
+  ];
```

Comment:
i think this is ok to merge now to get the framework of testing in, but we will want to map these urls automatically, eventually.

envisioning something like:

`const { htmlPaths } = require('../../tools/pl-paths.js');
const plRoot = 'http://0.0.0.0:8080/app-pl/pl/';
const scenarios = htmlPaths.map(partial => { label: partial, url: `${plRoot}/${partial}`, ...defaultScenarioSettings });
`

this will get every particle demo page mapped to our default scenario settings, so we don't have to keep adding manually as new components are made. projects will still have to set up additional scenarios manually for dev/staging/prod links, but particle would be covered automatically

Reply:
For the record, these comments have been implemented.